developer/system/template-manager.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: doc
+title: Template Manager
+permalink: /doc/template-manager/
+---
+
+Template Manager Design
+=======================
+
```

Comment:
I suggest adding a brief introduction before the first section (`Motivation`).

Reply:
Done.

Reply:
Thanks! I removed the `Introduction` heading, because (1) I believe it is superfluous, and (2) part of my intention (which I failed to make explicit) was to have some body text between the h1 and the first h2, since otherwise it looks rather bad.

Reply:
> a template manager application proposed in the 2020 Google Summer of Code program.

Is this intended to be a temporary bit of text that will be removed later? Inevitably, people will read this in the future, then ask us, "When will the Template Manager be updated and ready for real-world use? The documentation says it's  just some GSoC project from back in 2020!"

Reply:
I see. My intent was simply to mention the historical context in which the project was introduced, but I get the concern.

Perhaps it can either be changed to "originally proposed" or be removed outright?

Reply:
The `Motivation` section seems like a better place for it.

Reply:
Fixed.

> Is this intended to be a temporary bit of text that will be removed later?

Speaking of this, should there be a piece of text that indicates that the application is not yet merged / does not exist in the stable release of Qubes?

Reply:
> Speaking of this, should there be a piece of text that indicates that the application is not yet merged / does not exist in the stable release of Qubes?

Yes, definitely.